perun/collect/memory/malloc.c

```diff
@@ -168,7 +168,8 @@ __attribute__ ((constructor)) void initialize (void) {
 __attribute__((destructor)) void finalize (void) {
     if(logFile != NULL){
         fprintf(logFile, "EXIT %fs\n", clock() / (double)CLOCKS_PER_SEC);
-        fclose(logFile);
+        // This is causing segfaults for some reason, hotfix
```

Comment:
```suggestion
        // FIXME: This is causing segfaults for some reason, hotfix
```
This way it won't go fully ingored.

Reply:
Fixed.

Reply:
I have updated the README instructions a bit. Most of the instructions remained as they were, as the installation is usually done through the Makefile, which was updated accordingly. However, I have removed the outdated `perun init` command in both install/dev, and included some basic Tox instructions.